docs/index.rst

```diff
@@ -1,10 +1,13 @@
-.. Nuts discovery documentation master file, created by
-   sphinx-quickstart on Fri Feb  8 14:20:59 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. _nuts-discovery:
 
-Welcome to Nuts discovery's documentation!
-==========================================
+Nuts Discovery
+==============
```

Comment:
This might be a nice place to explain what the discovery service is.

Reply:
The general explanation is in the main docs. We can duplicate that?